Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -224,12 +324,15 @@ def ip_command(client: Client, ip_address: str, ip_version: str) -> List[Command
                                     argument=ip_)
         if raw_response and raw_response != 404:
             ip_version = FeedIndicatorType.IP if ip_version == 'IPv4' else FeedIndicatorType.IPv6
-            relationships = create_attack_pattern_relationships(client, raw_response=raw_response,
-                                                                entity_a=ip_, entity_a_type=ip_version)
+            relationships = create_relationships(client, extract_attack_ids(raw_response), ip_, ip_version, 'display_name',
```

Comment:
Why in this case do we create a relationship OUTSIDE the relationship manager?

Reply:
Can you elaborate on why it is not related to the max_indicator_relationships?

Reply:
This relationship was implemented before I started working on the enhancement and shouldn't be limited by the *max_indicator_relationships* param (I just double-checked with Meital). 
Since the other relationships are created only if the *max_indicator_relationships* != 0 so I differed between this two rs creators.

Reply:
So the only difference is that it does not dependent on the max_indicator_relationships? If so, I don't see a reason why it shouldn't be part of the relationship manager outside the max_indicator_relationships if statement. It looks very confusing to have a relationship manager that does not manage all the relationships

Reply:
got it.